whois-commons/src/main/java/net/ripe/db/whois/common/sso/AuthServiceClient.java

```diff
@@ -281,4 +282,33 @@ public MemberContactsResponse getOrgsContactsGroupsEmails(final Long membershipI
         }
     }
 
+    @Nullable
+    @Stopwatch(thresholdMs = 100L)
+    public MemberContactsResponse getLirsAccounts(final List<String> lirIds) {
```

Comment:
rename to getMemberDetailsForLirs?

Reply:
Also do you think its better to return Map<Long, MemberContactsResponse> , As every call we need to filter out per memebershipId so why not return it from here only . Saves multiple times filtering it out

Reply:
The map is not with the MemberCnntactResponse
There could be "batch_amount" memberships in the MemberContactResponse
The Map is with the ContactDetails there the relation is 1 to 1.